src/msg/en.js

```diff
@@ -29,5 +29,7 @@ en.LOGOUT = "Logout";
 en.TUTOR = "Tutor";
 en.CERTIFICATE_COMPLETION_WARNING =
   "Please complete all the courses to unlock %1 certificate";
+  en.PEEPUL_CERTIFICATE_COMPLETE_WARNING =
+  "Please complete at least 80% of course to unlock %1 certificate";
```

Comment:
Why is it ` %1` certificate?

Reply:
%1 is for adding the name of pathway course for which the certificate needs to be provided